### ISSUE TITLE
Configure linux-aws with VHOST_NET

### ISSUE THREAD

**Issue by gtkramer** [clearlinux/distribution#1139]
Kata Containers cannot run in AWS on an i3.metal instance with Clear Linux OOB unless we first modprobe vhost_net.  Otherwise, the following error message is seen:

```
docker: Error response from daemon: OCI runtime create failed: Could
not setup vhost fds eth0 : open /dev/vhost-net: no such file or
directory: unknown."
```

Please consider adding this to the kernel configuration for linux-aws.

**Comment by miguelinux** [comment]
it should be fixed at **linux-aws-5.2.9-152**

**Comment by gtkramer** [comment]
@miguelinux confirmed fixed
